src/generate.ts

```diff
@@ -193,6 +195,7 @@ export function createGeneratePlugin({
                 .filter((v) => v.isEntry)
                 .map((v) => v.id)
           const options: Omit<TscOptions, 'programs'> = {
+            tsconfig,
```

Comment:
Remove `tsconfigDir`, since there's already `tsconfig`

Reply:
https://github.com/sxzz/rolldown-plugin-dts/blob/a0e77c173a6ab61ae5f536c959c841331226ccfa/src/index.ts#L180

Based on this logic, `tsconfig` can empty but `tsconfigDir` cannot. `tsconfigDir` will be `cwd` if `tsconfig` is empty. Therefore, I cannot infer `tsconfigDir` simply base on `tsconfig` in `utils/tsc.ts`.